scripts/template_upload_openshift.py

```diff
@@ -77,8 +77,8 @@ def list_templates(hostname, username, password, upload_folder):
             return []
 
 
-def upload_template(hostname, username, password,
-                    provider, url, name, provider_data, stream, upload_folder):
+def upload_template(hostname, username, password, provider, url, name, provider_data,
+                    stream, upload_folder, oc_username, oc_password):
```

Comment:
Can we pull these from `cfme.utils.conf` + `provider.data.<some key for creds>` instead of passing them?

Reply:
I see below where you are before calling this, nvm.